### ISSUE TITLE
How to understand feat_mlp, logits, centers_logits in 2D image network?

### ISSUE THREAD

**Issue by whuhxb** [FlamieZhu/Balanced-Contrastive-Learning#1]
Hi @FlamieZhu 

I‘m trying to apply the loss you proposed to 3D point cloud. I have one question. How to understand feat_mlp, logits, centers_logits in 2D image network? If I use this loss for 3D semantic segmentation, how to correspond them? Thanks a lot.

Xiaobing

**Comment by FlamieZhu** [comment]
Hi,
Both feat_mlp and centers_logits are designed to boost representation learning, and the final prediction depends only on logits. I'm not familiar with 3D segmentation, but SCL is only used to help learn a more balanced feature space and help get better predictions, and it can be discarded during testing.

**Comment by Serissa** [comment]
> Hi @FlamieZhu
> 
> I‘m trying to apply the loss you proposed to 3D point cloud. I have one question. How to understand feat_mlp, logits, centers_logits in 2D image network? If I use this loss for 3D semantic segmentation, how to correspond them? Thanks a lot.
> 
> I have seen that you used CE+SCL in training, but in testing just used CE. Could I still use CE+SCL in testing?
> 
> Xiaobing

Hi,
Is the 3D segmentation effect improved?

**Comment by whuhxb** [comment]
> > Hi @FlamieZhu
> > I‘m trying to apply the loss you proposed to 3D point cloud. I have one question. How to understand feat_mlp, logits, centers_logits in 2D image network? If I use this loss for 3D semantic segmentation, how to correspond them? Thanks a lot.
> > I have seen that you used CE+SCL in training, but in testing just used CE. Could I still use CE+SCL in testing?
> > Xiaobing
> 
> Hi, Is the 3D segmentation effect improved?

Still trying.
